test/e2e/webdriver/driver.js

```diff
@@ -589,6 +589,17 @@ class Driver {
     }
   }
 
+  async takeScreenshot(title) {
+    const artifactDir = `./test-artifacts/${this.browser}/${title}`;
```

Comment:
[nit] we can move this as a const at the top of the file since it's being used in another area of this test

nice method!